samples/bigquery_cmek.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPBigQueryCMEKEncryptionConstraintV1
+metadata:
+  name: gcp-bq-cmek-encryption-v1
+spec:
+  severity: high
+  match:
+    target: ["organization/*"]
+  parameters: {}
```

Comment:
Nit. Please add newline at file end.

Reply:
fixed here and in the original test constraint